README.md

```diff
@@ -105,8 +105,9 @@ that module's code for implementation details.
 ## Browser Support
 
 Because this module leverages _native_ `WebSockets`, the browser support for this
-module is limited to only those browsers which support native `WebSocket`. That
-typically means the last two major versions of a particular browser.
+module is limited to only those browsers which support native `WebSocket`. This
+includes all browser versions since 2013, including Internet Explorer 10 and 11.
+[https://caniuse.com/#feat=websockets](See here for a full compatibility table).
```

Comment:
> This includes all browser versions since 2013

That's factually inaccurate. There are a litany of browsers in China and India that have versions published since 2013 which don't support `WebSocket`, and there are great numbers of users that have to be on those browsers (the Vue.js crowd is a big segment). That's the primary reason that `webpack-serve` exists separately from `webpack-dev-server`.

> including Internet Explorer 10 and 11

[Microsoft no longer supports IE10](https://www.microsoft.com/en-us/windowsforbusiness/end-of-ie-support), and neither do `webpack-contrib` projects. Given that, the "last two browser versions" statement is still accurate and sets the proper expectation. 

I'm totally good adding that caniuse link, and it probably should have been in there from the start. I would like to change that though, to:

```
Please visit [caniuse.com](https://caniuse.com/#feat=websockets) for a full compatibility table.
```

(your link markdown was backwards :))

Reply:
I'd discounted Opera Mini thinking supported no JS at all. Today I learned it's just _extremely_ limited. Good to know.

Seems like the other browsers you're talking about aren't even visible on caniuse.com (I'd checked UC, QQ, Baidu), but I'll see if I can link to the page with all browsers selected so people get the best possible picture of compatibility worldwide.